src/functions/anti-raid/usernameCheck.ts

```diff
@@ -0,0 +1,52 @@
+const badUsernamesRegex = [
```

Comment:
Requiring this to be changed in code and redeployed in order to update the list seems unnecessarily challenging to use. I'd suggest building out a separate database schema to handle this so that it can be updated on the fly. In turn, this could enable commands to update the list and retroactively apply filters.

Reply:
Should I use Redis for it?

Reply:
I will use the `Hash` maps of Redis to get it done

Reply:
Implemented dynamic system